nativelib/src/main/scala/scala/scalanative/posix/sys/mman.scala

```diff
@@ -0,0 +1,132 @@
+package scala.scalanative
+package posix.sys
+
+import native._
+import types.{mode_t, off_t}
+
+@extern
+object mman {
```

Comment:
This bindings are missing some APIs with posix_ prefix as defined in http://pubs.opengroup.org/onlinepubs/9699919799/